### ISSUE TITLE
[Docs] REST and GraphQL API Reference updates needed for multi-context support

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#5091]
#### Desired State
1. https://docs.meshery.io/reference/rest-apis
1. https://docs.meshery.io/reference/graphql-apis

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by leecalcote** [comment]
@Revolyssup a bit of docs update needed.

**Comment by leecalcote** [comment]
@Revolyssup please triage the issue. Are you going to complete this or should it be reassigned?

**Comment by Revolyssup** [comment]
> @Revolyssup please triage the issue. Are you going to complete this or should it be reassigned?

Putting this under a clickup task to track there and will get back to this after the top priority items are done.

**Comment by leecalcote** [comment]
@Revolyssup this reminds an open item

**Comment by matrixkavi** [comment]
Is this issue still relevant?

**Comment by leecalcote** [comment]
No, it's not. Thanks so much for inquiring about this, @matrixkavi . 👍
